proposals/nnnn-actors.md

```diff
@@ -180,17 +172,18 @@ actor MonopolyAccount : BankAccount {
 
 Cross-actor references to an actor property are permitted as an asynchronous call so long as they are read-only accesses:
 
-
 ```swift
 func checkBalance(account: BankAccount) {
   print(await account.balance)   // okay
   await account.balance = 1000.0 // error: cross-actor property mutations are not permitted
 }
 ```
 
+> **Rationale**: it is possible to support cross-actor property sets. However, cross-actor `inout` operations cannot be reasonably supported because there would be an implicit suspension point between the "get" and the "set" that could introduce what would effectively be race conditions. Moreover, setting properties asynchronously may make it easier to break invariants unintentionally if, e.g., two properties need to be updated at once to maintain an invariant.
```

Comment:
👍 

Yeah, I would be really uncomfortable by people just setting individual properties on actors like that... That just would not be helpful to embrace to actor way of modeling applications -- ask the actor to change stuff, and make that hwoever coarse or fine grained it has to be, rather than exposing public settable things. 👍